draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -461,84 +446,55 @@ Coherent pluggable have revolutionized optical networking by offering a powerful
 
-* Method 1: Defined by a Standards Developing Organization (SDO) such as ITU-T
-* Method 2: Defined by a forum such as Optical Internetworking Forum (OIF), a group of operators (OpenConfig) or by an individual vendor
-
-These methods define a namespace for the authority that specified the particular supported mode, whether it be a standards body, an association, a group of operators, or an individual vendor. For example, the supported modes in method 1 are well-known capabilities established by standards bodies such as the ITU-T {{G.698.2}}. These modes are recognized and supported by coherent pluggables, routers, and SDN controllers. This draft also supports the work of other Standards Development Organizations (SDOs), forums, or individual vendors as their specifications become available. In contrast, the supported-modes in method 2 are capabilities specified by forums such as OIF {{OIF-400ZR}}, OpenConfig, or by individual vendors. These supported-modes might not be supported by all coherent-pluggables, routers or SDN controllers. In specific, the supported-modes defined by an individual vendor might be not be recognized and supported by any routers and SDN controllers.
+This current draft adheres to the definitions provided in draft {{ietf-impairment-yang}}. Specifically, Section 2.6 of draft {{ietf-impairment-yang}} outlines three distinct approaches for describing "supported mode" capabilities:
 
-It is important to note that, from a functional perspective of coherent pluggables, the authority that defined the supported modes is less critical than the specific attributes associated with each supported mode. The key consideration is the list of attributes defined by these supported modes.
+* Standard Mode: This mode pertains to optical specifications developed by standards development organizations (SDOs), such as the ITU-T recommendation {{G.698.2}}.
+* Organizational Mode: In this mode, optical interface specifications are defined by operators, industry forums (e.g., Optical Internetworking Forum (OIF) or OpenConfig), or equipment vendors. This allows for the utilization of optical pluggable capabilities that extend beyond existing standards.
+* Explicit Mode: This mode enables the explicit encoding of any subset of parameters (e.g., FEC type, modulation type) to facilitate interoperability checks by a controller entity through means not covered within this draft.
 
-As illustrated in {{figure-plug-supported-mode}}, this document utilizes a consistent data structure for defining supported modes, irrespective of the authority that specified them. It includes:
+For more detailed information, please refer to draft {{ietf-impairment-yang}}.
 
-* supported-mode-id: Identification of the supported mode
-* organization-id: This provides a name space for an authority who defines the supported-mode, i.e., the authority that defines the supported-mode, such as ITU-T, OIF, OpenConfig, or a vendor
```

Comment:
ITU-T defined the application code which is a particular branch. It is confusing (but not wrong) to add it to the list of organizational modes. Suggest to remove it from the list of examples.